articles/openshift/support-lifecycle.md

```diff
@@ -10,125 +10,113 @@ ms.date: 12/18/2023
 
-Upgrade channels are tied to a minor version of Red Hat OpenShift Container Platform (OCP). For instance, OCP 4.9 upgrade never includes an upgrade to a 4.10 release. Upgrade channels control only release selection and don't impact the version of the cluster.
-
-Azure Red Hat OpenShift 4 supports stable channels only. For example: stable-4.9.
-
-You can use the stable-4.10 channel to upgrade from a previous minor version of Azure Red Hat OpenShift. Clusters upgraded using fast, prerelease, and candidate channels aren't supported.
-
-If you change to a channel that doesn't include your current release, an alert displays and no updates can be recommended. However, you can safely change back to your original channel at any point.
+> [!IMPORTANT]
+> Azure Red Hat OpenShift provides support for stable channels only. For example: `stable-4.15`.
+>
 
-## Red Hat OpenShift Container Platform version support policy
+You can use the `stable-4.15` channel to update from a previous minor version of Azure Red Hat OpenShift. Clusters updated using `fast` or `candidate` channels could put your cluster in a [Limited Support state](#limited-support-status).
 
-Azure Red Hat OpenShift supports two generally available (GA) minor versions of Red Hat OpenShift Container Platform:
-* The latest GA minor version that is released in Azure Red Hat OpenShift (referred to as N)
+## Azure Red Hat OpenShift version support policy
 
-* One previous minor version (N-1)
+### Azure Red Hat OpenShift version availability
 
-If available in a stable upgrade channel, newer minor releases (N+1, N+2) available in upstream OCP are supported as well.
+An Azure Red Hat OpenShift release is available through one of two mechanisms: 
+* When an update to a newer version is available for an existing cluster
+* When a new version is available as an install target for a new cluster
 
-Critical patch updates are applied to clusters automatically by Azure Red Hat OpenShift Site Reliability Engineers (SRE). Customers that wish to install patch updates in advance are free to do so.
+#### Update availability
+Azure Red Hat OpenShift supports generally available (GA) minor versions of Red Hat OpenShift Container Platform from when an update is available in the OpenShift `stable` channel.  Update availability can be checked at the following page, [Red Hat OpenShift Container Platform Update Graph](https://access.redhat.com/labs/ocpupgradegraph/update_path). 
 
-For example, if Azure Red Hat OpenShift introduces 4.10.z today, support is provided for the following versions:
+#### Install availability
+Installable versions can be validated by using the [Azure Red Hat OpenShift release calendar](#azure-red-hat-openshift-release-calendar) below or by running the following command: 
```

Comment:
Should change this to "running the following Azure CLI command:" with Azure CLI being a link to the Azure CLI page.

Reply:
Addded.